src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -83,6 +83,14 @@ public TimeSpan UserProcessorTime
             }
         }
 
+        public int HandleCount
+        {
+            get
+            {
+                return Directory.GetDirectories("/proc/" + Id + "/fd", "*", SearchOption.TopDirectoryOnly).Length;
```

Comment:
We need to be careful about exceptions.  for example, it's quite possible the target process will have gone away by the time this is called, in which case this will throw an exception.